<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ui_problem_to_solve_tooltip_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Problem to solve: </string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="ui_central_graphics_area_graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_skip_to_start_pushButton">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>430</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/skip_to_begin.svg</normaloff>:/icons/icons/skip_to_begin.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>42</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_skip_back_one_step_pushButton">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>430</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/step_backwards.svg</normaloff>:/icons/icons/step_backwards.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_skip_forward_one_step_pushButton">
    <property name="geometry">
     <rect>
      <x>445</x>
      <y>430</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/step_forwards.svg</normaloff>:/icons/icons/step_forwards.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_skip_to_end_pushButton">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>430</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/skip_to_end.svg</normaloff>:/icons/icons/skip_to_end.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>42</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="ui_take_string_to_test_textEdit">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>460</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>2</number>
    </property>
    <property name="placeholderText">
     <string>Enter a string to test</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui_live_input_proc_summary_label">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>490</y>
      <width>130</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TK: live summary</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui_input_accepted_or_rejected_label">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>530</y>
      <width>130</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TK: accepted or rejected</string>
    </property>
   </widget>
   <widget class="QLabel" name="ui_problem_to_solve_label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TK: Problem to solve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_difficulty_easy_pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Easy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_difficulty_medium_pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Medium</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_difficulty_hard_pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Hard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_toggle_solution_pushButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>510</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle solution</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_submit_solution_pushButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>470</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ui_request_hint_pushButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>430</y>
      <width>110</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Hint</string>
    </property>
   </widget>
   <zorder>ui_central_graphics_area_graphicsView</zorder>
   <zorder>ui_problem_to_solve_tooltip_label</zorder>
   <zorder>ui_skip_to_start_pushButton</zorder>
   <zorder>ui_skip_back_one_step_pushButton</zorder>
   <zorder>ui_skip_forward_one_step_pushButton</zorder>
   <zorder>ui_skip_to_end_pushButton</zorder>
   <zorder>ui_take_string_to_test_textEdit</zorder>
   <zorder>ui_live_input_proc_summary_label</zorder>
   <zorder>ui_input_accepted_or_rejected_label</zorder>
   <zorder>ui_problem_to_solve_label</zorder>
   <zorder>ui_difficulty_easy_pushButton</zorder>
   <zorder>ui_difficulty_medium_pushButton</zorder>
   <zorder>ui_difficulty_hard_pushButton</zorder>
   <zorder>ui_toggle_solution_pushButton</zorder>
   <zorder>ui_submit_solution_pushButton</zorder>
   <zorder>ui_request_hint_pushButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
